api/vpc_gcp_peering_temp.go

```diff
@@ -0,0 +1,125 @@
+package api
+
+// Temporary functions between versions in order to change managed VPC peering.
+// Instead of using instanceID as identifier, use managed vpcID as identifier.
```

Comment:
I think we should avoid calling this "Temporary"

Instead of suffixing the methods with `Temp`, can we use `WithVpcId`?

Reply:
Yes, will change the suffix.